common/src/main/java/com/microsoft/identity/common/internal/cache/MsalOAuth2TokenCache.java

```diff
@@ -184,17 +263,118 @@ private boolean scopesIntersect(final AccessToken token1, final AccessToken toke
         return scopeSet;
     }
 
-    private void saveAccount(
-            final OAuth2Strategy oAuth2Strategy,
-            final AuthorizationRequest request,
-            final TokenResponse response) {
-        final String methodName = "saveAccount";
-        Logger.entering(TAG, methodName, oAuth2Strategy, request, response);
-        final Account accountToSave = mAccountCredentialAdapter.createAccount(oAuth2Strategy, request, response);
-        Logger.info(TAG + ":" + methodName, "Saving Account...");
-        Logger.infoPII(TAG + ":" + methodName, "Account: " + accountToSave);
-        mAccountCredentialCache.saveAccount(accountToSave);
-        Logger.exiting(TAG, methodName);
+    private static boolean isSchemaCompliant(final Class<?> clazz, final String[][] params) {
```

Comment:
Borrowing from the exception chapter that @shoatman shared, should we throw if its not compliant?

Reply:
This is a helper method for `validateCacheArtifacts()` -- the Exception is thrown there :)